client/components/Home/styles.ts

```diff
@@ -1,116 +1,48 @@
 import { makeStyles } from "@material-ui/core/styles";
 
-export const useHomeStyles = makeStyles((theme) => ({
-  container: {
-    padding: theme.spacing(4)
-  },
-  header: {
-    marginBottom: theme.spacing(3)
-  },
-  mainContent: {
-    paddingBottom: theme.spacing(2),
-    display: "flex",
-    height: "60%"
-  },
-  feedContainer: {
-    flex: 2,
-    height: "100%"
-  },
-  filmContainer: {
-    flex: 1,
-    height: "100%",
-    marginLeft: theme.spacing(2)
-  }
-}));
-
-export const useFeedStyles = makeStyles((theme) => ({
-  feedContainer: {
-    height: "100%",
-    padding: theme.spacing(2)
-  },
-  recentFeedContainer: {
-    display: "flex",
-    padding: theme.spacing(1)
-  },
-  infos: {
-    marginLeft: theme.spacing(1)
-  },
-  header: {
-    marginBottom: theme.spacing(1),
-    marginLeft: theme.spacing(1)
-  }
-}));
-
-export const useRecentMoviesStyle = makeStyles((theme) => ({
-  container: {
-    padding: theme.spacing(3)
-  },
-  recentFilmContainer: {
-    display: "flex",
-    justifyContent: "space-evenly",
-    padding: theme.spacing(3)
-  },
-  recentFilm: {
-    position: "relative",
-    transition: "transform 0.2s",
-    "&:hover": {
-      transform: "scale(1.15)"
-    }
-  },
-  recentFilmTitle: {
-    color: theme.palette.text.primary,
-    userSelect: "none",
-    display: "flex",
-    justifyContent: "center",
-    alignItems: "center",
-    position: "absolute",
-    top: 0,
-    left: 0,
-    height: "100%",
+export const useHomeStyles = makeStyles(() => ({
+  mainPoster: {
+    background:
+      "linear-gradient(to bottom, transparent 40%, black 80%), url('https://fr.web.img6.acsta.net/pictures/19/11/19/12/39/2279585.jpg')",
```

Comment:
Using a image not on our server can be dangerous (the image can disappear, change...). You should download the image and put it in the `/public` folder.
You can then access it here: `localhost:8080/public/myImageName.png`